.github/workflows/e2e.yaml

```diff
@@ -19,10 +19,6 @@ jobs:
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go-
-      - name: Setup Go
```

Comment:
How will GitHub cache the go modules if we write them inside the container?

Reply:
I'm trying to figure out all the details now, but I assume we're gonna need to mount the volume.